pentaho-pdi-platform/src/main/java/org/pentaho/platform/pdi/WebContextServlet.java

```diff
@@ -85,6 +99,36 @@
 
   }
 
+  private String getRequireCfg() throws IOException {
+    Integer waitTime = null;
+
+    if ( cache != null ) {
```

Comment:
Is it worth it to be adding a cache just to cache the WaitSeconds? Did you find any performance issues on getting this value?